tools/processresults.py

```diff
@@ -11,13 +11,18 @@
 import re
 
 DOGFOOD_START_DATE = datetime(2014, 12, 01, 0, 0, 0, 0)
+DEMOGRAPHIC_STABLE_DATE = datetime(2014, 12, 18, 0, 0, 0, 0)
+DEMOGRAPHIC_CSV_PREFIX = 'demographics'
 CONDITIONS = [
     'ssl-overridable-proceed', 'ssl-overridable-noproceed',
     'ssl-nonoverridable', 'malware-proceed', 'malware-noproceed',
     'phishing-proceed', 'phishing-noproceed', 'extension-proceed',
     'extension-noproceed']
+TECHFAMILIAR_QUESTION_PREFIX = ('How familiar are you with each of the '
```

Comment:
This seems to be conceptually different from the rest of the PR which (by the description) is about adding IDKs